lua/astrocommunity/utility/snacks-nvim/init.lua

```diff
@@ -0,0 +1,109 @@
+return {
+  "folke/snacks.nvim",
+  priority = 10000,
+  lazy = false,
+  ---@param opts snacks.Config
+  opts = function(_, opts)
+    local astrocore = require "astrocore"
+    return astrocore.extend_tbl(opts, {
```

Comment:
I think those options would make snacks windows fall in line with other current float windows style
```suggestion
    return astrocore.extend_tbl(opts, {
    win = {
      border = "rounded",
      backdrop = false,
    },
```